Flaky test: TestContainerPsContext ?

Flaky test on WindowsTP4? https://jenkins.dockerproject.org/job/Docker-PRs-WoW-TP4/2071/console

Seen on https://github.com/docker/docker/pull/20903

```
02:55:41 ---> Making bundle: test-unit (in bundles/1.11.0-dev/test-unit)
02:55:41 Fri Mar  4 02:55:41 CUT 2016
02:56:02 ok  	github.com/docker/docker/api	0.284s	coverage: 92.5% of statements
02:56:21 ok  	github.com/docker/docker/api/client	0.098s	coverage: 1.3% of statements
02:56:21 --- FAIL: TestContainerPsContext (0.31s)
02:56:21 	custom_test.go:67: Expected Less than a second, was 1 seconds
02:56:21 FAIL
02:56:21 coverage: 90.6% of statements
```

--- a/api/client/formatter/custom_test.go
+++ b/api/client/formatter/custom_test.go
@@ -12,7 +12,7 @@ import (
 
 func TestContainerPsContext(t *testing.T) {
 	containerID := stringid.GenerateRandomID()
-	unix := time.Now().Unix()
+	unix := time.Now().Add(-65 * time.Second).Unix()
 
 	var ctx containerContext
 	cases := []struct {
@@ -55,7 +55,7 @@ func TestContainerPsContext(t *testing.T) {
 		{types.Container{SizeRw: 10, SizeRootFs: 20}, true, "10 B (virtual 20 B)", sizeHeader, ctx.Size},
 		{types.Container{}, true, "", labelsHeader, ctx.Labels},
 		{types.Container{Labels: map[string]string{"cpu": "6", "storage": "ssd"}}, true, "cpu=6,storage=ssd", labelsHeader, ctx.Labels},
-		{types.Container{Created: unix}, true, "Less than a second", runningForHeader, ctx.RunningFor},
+		{types.Container{Created: unix}, true, "About a minute", runningForHeader, ctx.RunningFor},
 	}
 
 	for _, c := range cases {

